packages/api-server/src/index.ts

```diff
@@ -11,9 +11,14 @@ if (process.argv0 === 'api-server') {
   console.log()
 }
 
-const { port, socket } = yargs
+const { port, socket, routePrefix } = yargs
   .option('port', { default: 8911, type: 'number', alias: 'p' })
-  .option('socket', { type: 'string' }).argv
+  .option('socket', { type: 'string' })
+  .option('routePrefix', {
```

Comment:
I'm wonder if we could name this `root-path` instead?

Reply:
Agreed. I'll have another swing at it, combining it with the coerce feedback. Thanks!